src/controllers/users.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from 'express';
+
+import db from '../db';
+import { CreateUserRequest, DeleteUserRequest, GetUserRequest, UpdateUserRequest, User } from '../types/user';
```

Comment:
Use `import type`. This indicates they are type-only imports, making it clear that these imports are not part of the JavaScript runtime. It also helps bundlers and compilers with optimizations like tree shaking, ensuring that these type definitions, which are unnecessary at runtime